sdk-actors/pom.xml

```diff
@@ -7,12 +7,12 @@
   <parent>
     <groupId>io.dapr</groupId>
     <artifactId>dapr-sdk-parent</artifactId>
-    <version>0.2.0</version>
+    <version>0.2.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>dapr-sdk-actors</artifactId>
   <packaging>jar</packaging>
-  <version>0.2.0</version>
+  <version>0.2.0-SNAPSHOT</version>
```

Comment:
Can all packages refer to a property from parent to define version?

Reply:
Nope. child pom uses 3 properties(artifactId, packaging, version) to find the parent pom.